components/camel-jetty-common/src/main/java/org/apache/camel/component/jetty/JettyHttpComponent.java

```diff
@@ -1016,6 +1028,46 @@ public void setSendServerVersion(boolean sendServerVersion) {
         this.sendServerVersion = sendServerVersion;
     }
 
+    public long getMaxFileSize() {
+        return maxFileSize;
+    }
+
+    @Metadata(description = "The maximum size allowed for uploaded files. -1 means no limit",
+              defaultValue = "-1", label = "consumer")
+    public void setMaxFileSize(long maxFileSize) {
+        this.maxFileSize = maxFileSize;
+    }
+
+    public long getMaxRequestSize() {
+        return maxRequestSize;
+    }
+
+    @Metadata(description = "The maximum size allowed for multipart/form-data requests. -1 means no limit",
+              defaultValue = "-1", label = "consumer")
```

Comment:
Maybe these should also be advanced label like in the endpoint

Reply:
It makes sense indeed